### PR TITLE
DSP-604 Update general contribution information

### PR DESCRIPTION
**Pull request by kilchenmann**
New page about DSP developer contribution in general. All main DSP relevant software and repositories has the same workflow on Github. This new page describes 

- How to contribute in general
- How to do git commits
- How to do a PR incl. labels --> resolves [DSP-604](https://dasch.myjetbrains.com/youtrack/issue/DSP-604)
- How the Github actions workflows (CI) generates automated stuff
- Additional I added some badges to see all repository status all at once:
  - Github repo status and version tag
  - Github actions workflow status
  - Docker image or npm package link incl. latest version

### CODE REVIEW

File: docs/developers/dsp/contribution/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+After each push into the master branch &mdash; after each merge from a pull request &mdash; the release notes for the next release are updated. This release called "Next release" is a draft and can be used to publish the [real release](#release) later.
+
+The GitHub action we use for this step is [release-drafter](https://github.com/marketplace/actions/release-drafter).
+
+> This feature is not yet implemented in all repositories. We are still testing it in [DSP-APP](https://github.com/dasch-swiss/dsp-app/releases).
+
+### Release
+
+To make a real release, we have to publish the [release draft](#release-nots), mentioned above, manually. Update the tag and the release title with the release version number, including the prefix `v`: e.g. v1.0.0 or v1.0.0-rc.0
```
Comment (flavens) [inline docs/developers/dsp/contribution/index.md:135]: is #release-nots correct? the link is not broken but I do not think it leads to the correct page

File: docs/developers/dsp/contribution/index.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# How to contribute to the development of the DSP
```
Comment (flavens) [inline docs/developers/dsp/contribution/index.md:1]: I change the title but if you do not like it, you can reverse it

File: docs/developers/dsp/contribution/index.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+Resolves [<DSP-nr>](https://dasch.myjetbrains.com/youtrack/issue/<DSP-nr>)
+```
+
+Replace the `<DSP-nr>` with the real number e.g. `DSP-42`.
+
+### Add a label
+
+Add at least one of the corresponding labels to your PR:
+
+- **breaking** Breaking Changes
```
Comment (flavens) [inline docs/developers/dsp/contribution/index.md:85]: the label release is missing, should we add it?

**Review by flavens — APPROVED** [review]
LGTM - see my comments and questions

File: docs/developers/dsp/contribution/index.md (hunk 3 above)
Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md:85]: Ah yes, I wasn't sure about that. When do we need it? In the workflow process we will not have a specific PR for the release anymore. And we do not use it in the release template of release-drafter.

File: docs/developers/dsp/contribution/index.md (hunk 1 above)
Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md:135]: 😳 oups...ehm no, it should be #release-notes to link to the title correctly. I'll fix it...

Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md]: f12e360

**Pull request merged**